### ISSUE TITLE
Mis-recognize of potential form of 来る

### ISSUE THREAD

**Issue by SaltfishAmi** [birtles/rikaichamp#301]
![Untitled](https://user-images.githubusercontent.com/14184974/85041832-093ab880-b1bd-11ea-8291-3845eff32e3b.png)

The potential form of 来（く）る is 来（こ）られる or (while informal) 来（こ）れる, while Rikaichamp also recognizes くれる as one of the potential forms of 来る.
In this pic, 生まれてくれる is recognized as 生まれてくる 's potential form, which is even more confusing...

**Comment by birtles** [comment]
Yeah, 来る is tricky. Currently the rule for converting れる to る accepts vk verbs because we want to convert 来れる to 来る.

There are quite a few vk verbs but they nearly always end in 来る or くる so I think we could make that rule _not_ accept vk verbs and add a separate rule that matches on 来られる/来れる, i.e. including the kanji (we have separate rules for これる and こられる etc. already). Plus possibly 來る.

I notice we also treat くさせる as causative for 来る.
